src/main/scala/com/smartystreets/api/SmartyStreets.scala

```diff
@@ -11,7 +11,7 @@ class SmartyStreets private(authId: String,
                             authToken: String,
                             val groupSize: Int = 100) {
   private[api] lazy val client = HttpClient(HttpClientConfig(
-    retries = 100,
+    retries = 1,
```

Comment:
It might be a better idea to make this configurable in the construction of the SmartyStreets options so it can be configured by anyone using it.